src/test/java/pages/PlaylistActions.java

```diff
@@ -0,0 +1,46 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+public class PlaylistActions extends BasePage {
+
+    // Constructor
+    public PlaylistActions(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+    // LOCATORS
+    By clickPlaylist = By.xpath("//*[@id='playlists']/ul/li[3]/a");
+    By deletePlaylistAction = By.xpath("//*[@class='del btn-delete-playlist']");
+    By successPopUp = By.xpath("//*/div[2]/div[@class='success show']");
```

Comment:
if you are beginner does not know who to write the xpath then you can relay on the xpath that will be provided by chrome browser. Right click on element that you are interested in then click on Inspect. Now right click on the html code of the element Copy > Copy Xpath. Now you have the xpath in the clipboard for you.

_For example, here is the locator (Copy > Copy Xpath):_  `//*[@id="sidebar"]/section[1]/ul/li[3]/a`

**BUT I recommend not using plugins or copying the xpath from the page code using a browser or web developer.**
For example, here is the correct locator: ` //a[@class='songs']`

It is important to mention that the following principle must be held in deciding locators: the locator must be

- unique
- readable 
- maintainable

**HINT**

**You can use this extension for easy locator search.**

 Ranorex Selocity

Ranorex Selocity is a free extension for Chrome DevTools. It allows you to create unique selectors through a smart selector generator. Feedback is created, allowing you to locate your web page's elements in the most secure manner. https://www.ranorex.com/selocity/browser-extension/

For example 

![image](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/104761482/d7f9b33c-8692-4fd8-b680-fb0da31ffd7b)

Xpath cheatsheet https://devhints.io/xpath